src/gluonts/dataset/artificial/_base.py

```diff
@@ -848,3 +848,26 @@ def constant_dataset() -> Tuple[DatasetInfo, Dataset, Dataset]:
     )
 
     return info, train_ds, test_ds
+
+
+def shuffle_testing_dataset(n):
```

Comment:
Can this be moved directly in the test script?

Reply:
just removed this